extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/ContainerImageJibConfig.java

```diff
@@ -33,7 +33,7 @@ public interface ContainerImageJibConfig {
      * "registry.access.redhat.com/ubi9/ubi-minimal" which is a bigger base image, but provide more built-in utilities
      * such as the microdnf package manager.
      */
-    @WithDefault(ContainerImages.QUARKUS_MICRO_IMAGE)
```

Comment:
The change I proposed above should also avoid changing the default value here often